src/teleop_twist_joy.cpp

```diff
@@ -80,7 +80,7 @@ TeleopTwistJoy::TeleopTwistJoy(rclcpp::node::Node::SharedPtr & node)
   rmw_qos_profile_t custom_qos_profile;
   custom_qos_profile.history = RMW_QOS_POLICY_HISTORY_KEEP_LAST;
   custom_qos_profile.depth = 50;
-  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT;
```

Comment:
For which topics is that not working ? the `cmd_vel` topic that goes to the kobuki base or the `joy` topic from the joystick ? (I'd assume it's the cmd_vel given that you had the issue of spin not exiting in that kobuki_node)